portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
+    String textValue = request.getParameter("text-input");
+
+    // Print the value so you can see it in the server logs.
+    System.out.println("You submitted: " + textValue);
+
+    // Write the value to the response so the user can see it.
+    response.getWriter().println("You submitted: " + textValue);
+    
+    response.sendRedirect("https://nnietovega-sps-spring21.uc.r.appspot.com/");
```

Comment:
Ideally, this code wouldn't include the hostname of the application. That way, when it's deployed to another project or run locally, it continues to work as expected.

Looking at the docs for the `sendRedirect` method, it appears that you might be able to call it with just the part of the path after the hostname and it will still work. Here, that might look like `response.sendRedirect("/");`

Reply:
Oh... I see. Thank you for your feedback. I'll change it and upload again :)